pkg/acsengine/defaults.go

```diff
@@ -804,7 +834,7 @@ func certsAlreadyPresent(c *api.CertificateProfile, m int) map[string]bool {
 }
 
 // getFirstConsecutiveStaticIPAddress returns the first static IP address of the given subnet.
-func getFirstConsecutiveStaticIPAddress(subnetStr string) string {
+func getFirstConsecutiveStaticIPAddress(subnetStr string, isVMSS bool) string {
```

Comment:
nit: we could make this func a method of `MasterProfile`, which would enable it to know whether or not it was configured as VMAS or VMSS